commons/addons/elastic.md

```diff
@@ -57,7 +57,12 @@ Learn more on [APM official documentation](https://www.elastic.co/guide/en/apm/g
 
 ### How to setup APM
 
-Any applications linked to the APM application will have the right credentials and APM endpoint automatically avaiable as environment variables. These variables can be picked up automatically by the APM agent you are using in your application. As such everything should work automatically.
+Any applications linked to the APM application will have the right credentials and APM endpoint automatically available as environment variables. These variables can be picked up automatically by the APM agent you are using in your application. As such everything should work automatically.
```

Comment:
not always true (e.g: https://www.elastic.co/guide/en/apm/agent/ruby/3.x/getting-started-rails.html#getting-started-rails)

and I would add this link: https://www.elastic.co/guide/en/apm/get-started/current/agents.html